src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.draftstore;
+
+import io.restassured.RestAssured;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.hamcrest.Matchers.equalTo;
+
+@RunWith(SpringRunner.class)
+public class ApplicationSmokeTest {
+
+    @Value("${test-url:http://localhost:8800}")
```

Comment:
can the default go to config file instead?

Reply:
Yes, that's the right place! Sorry, copied it too blindly